### PR TITLE
Update Taxi Fare tutorial to v 1.0.0

### PR DESCRIPTION
**Pull request by JRAlexander**
[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/taxi-fare?branch=pr-en-us-12035)

[Related Code Sample](https://github.com/dotnet/samples/pull/838)

#AB1484922

### CODE REVIEW

File: docs/machine-learning/tutorials/taxi-fare.md [hunk 1]
```diff
@@ -110,24 +98,23 @@ Add the following code right above the `Main` method to specify those paths and
 
 [!code-csharp[InitializePaths](~/samples/machine-learning/tutorials/TaxiFarePrediction/Program.cs#2 "Define variables to store the data file paths")]
 
-When building a model with ML.NET you start by creating an ML Context. This is comparable conceptually to using `DbContext` in Entity Framework. The environment provides a context for your machine learning job that can be used for exception tracking and logging.
+All ML.NET operations start in the [MLContext class](xref:Microsoft.ML.MLContext). Initializing `mlContext` creates a new ML.NET environment that can be shared across the model creation workflow objects. It's similar, conceptually, to `DBContext` in Entity Framework.
 
 ### Initialize variables in Main
 
-Create a variable called `mlContext` and initialize it with a new instance of `MLContext`.  Replace the `Console.WriteLine("Hello World!")` line with the following code in the `Main` method:
+Replace the `Console.WriteLine("Hello World!")` line in the `Main` method with the following code to declare and initialize the mlContext variable:
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/taxi-fare.md]: code fence `mlContext`

File: docs/machine-learning/tutorials/taxi-fare.md [hunk 2]
```diff
@@ -292,22 +241,22 @@ Add a call to the new method from the `Main` method, right under the `Evaluate`
...
 
 [!code-csharp[PredictionData](~/samples/machine-learning/tutorials/TaxiFarePrediction/Program.cs#23 "Create test data for single prediction")]
 
-We can use that to predict the fare based on a single instance of the taxi trip data. To get a prediction, use <xref:Microsoft.ML.PredictionEngine%602.Predict%2A> on the data. Note that the input data is a string and the model includes the featurization. Your pipeline is in sync during training and prediction. You didn’t have to write preprocessing/featurization code specifically for predictions, and the same API takes care of both batch and one-time predictions.
+Next, predict the fare based on a single instance of the taxi trip data and pass it to the `PredictionEngine` by adding the following as the next lines of code in the `TestSinglePrediction()` method:
 
 [!code-csharp[Predict](~/samples/machine-learning/tutorials/TaxiFarePrediction/Program.cs#24 "Create a prediction of taxi fare")]
 
+The [Predict()](xref:Microsoft.ML.PredictionEngine%602.Predict%2A) function makes a prediction on a single column of data.
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/taxi-fare.md]: Suggestion: change column to instance or sample

**Review by luisquintanilla — CHANGES_REQUESTED** [review]
Looks good @JRAlexander . Minor changes

**Review by luisquintanilla — APPROVED** [review]

**Comment by JRAlexander** [comment]
Thanks, @luisquintanilla!

**Pull request merged**